modules/openldap/go.mod

```diff
@@ -0,0 +1,57 @@
+module github.com/testcontainers/testcontainers-go/modules/openldap
```

Comment:
I think you need to run mod tidy here with the recent bumps. I know, we are in a monorepo and dependabot should automate the bumps across the repo 😞

Reply:
Done